code/game/objects/items/explosives/mine.dm

```diff
@@ -293,6 +384,76 @@
 	iff_signal = null
 
 
+// Subtype from this to create buried mines.
+/obj/item/explosive/mine/bury
+	use_tripwire = FALSE
+	needs_digging = TRUE
+	map_prespawn = FALSE
+	buried = FALSE
+	arming_time = 3 SECONDS
+	use_dir = FALSE
+	unacidable = TRUE
+	var/datum/effect_system/spark_spread/sparks = new
+
+/obj/item/explosive/mine/bury/examine(mob/user)
+	. = ..()
+	if(!buried)
+		. +=  "\n A small label on the bottom reads: 'To deploy: simply pull the pin to activate it and dig it in with your standard issue e-tool. This munition will automatically arm in [arming_time] seconds after being buried.' "
+	else
+		. +=  SPAN_DANGER("\n This unit is armed and ready")
+
+/obj/item/explosive/mine/bury/Destroy()
+	. = ..()
+	QDEL_NULL(sparks)
+
+/obj/item/explosive/mine/bury/Initialize(mapload, ...)
+	. = ..()
+	sparks.set_up(5, 0, src)
+	sparks.attach(src)
+
+/obj/item/explosive/mine/bury/disarm()
+	. = ..()
+	QDEL_NULL(sparks)
+
+
+/obj/item/explosive/mine/bury/prime()
+	. = ..()
+	QDEL_NULL(sparks)
+
+/obj/item/explosive/mine/bury/antitank
+	name = "\improper M19 Anti-Tank Mine"
+	desc = "This older anti tank mine design from the 21st century was rolled back into service simply due to the currently-used M307 EMP anti tank mines being unable to trigger against the minimally armored vehicles commonly used by CLF. Featuring a 250 pound minimum detonation threshold, it can be employed against all but the lightest of vehicles. Despite being outdated, it can still pack a punch against APCs and lighter vehicles, while its plastic construction prevents detection by simple methods."
+	icon_state = "antitank_mine"
+	w_class = SIZE_LARGE
+	layer = MOB_LAYER - 0.1 //You can't just randomly hide claymores under boxes. Booby-trapping bodies is fine though
+	explosive_power = 150
+	heavy_trigger = TRUE
+
+/obj/item/explosive/mine/bury/antitank/prime()
+	set waitfor = 0
+	create_shrapnel(loc, shrapnel_count, , ,/datum/ammo/bullet/shrapnel, cause_data)
+	cell_explosion(loc, explosive_power, 25, EXPLOSION_FALLOFF_SHAPE_EXPONENTIAL_HALF, dir, cause_data)
+	for(var/mob/living/carbon/M in oview(1, src))
+		M.AdjustStun(4)
+		M.KnockDown(4)
+		to_chat(M, SPAN_HIGHDANGER("Molten copper rips through your lower body!"))
+		M.apply_damage(50,BURN)
+		if(ishuman(M))
+			sparks.start()
+			var/mob/living/carbon/human/H = M
+			var/obj/limb/L = H.get_limb("l_leg")
+			var/obj/limb/R = H.get_limb("r_leg")
+			R.droplimb()
+			L.droplimb()
+			playsound(M.loc, "bone_break", 45, TRUE)
+			playsound(M.loc, "bone_break", 45, TRUE)
+	for(var/mob/living/carbon/M in view())
+		if(M && M.client)
+			shake_camera(M, 10, 1)
```

Comment:
Don't use view() etc without arguments, it's `usr` dependent which depends on who triggered the action rather than the distance to object

view() = what the claymore can see (including all clutter objects)
viewers() = what mobs can see the claymore

```suggestion
	for(var/mob/living/living_mob in viewers(7, src))
		if(living_mob.client)
			shake_camera(living_mob, 10, 1)
```

you can drop living and use as anything if you want to shake ghosts too which is probably funny and might not be a big deal